app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PostsHelper
+end
```

Comment:
- [ ] It seems that you used `rails generate scaffold`, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Project%20requirements-,Important%3A%20do%20NOT%20use%20rails%20generate%20scaffold%20command.,-The%20point%20of), Please remove the modules inside `helpers` directory because it's generated by the mentioned command.

![image](https://github.com/NoelLincoln/blog-app/assets/68796232/346f5316-58ee-4199-8d65-19b956ff4803)

Reply:
Thank you, I had missed that

Reply:
- [ ] @NoelLincoln As previously requested kindly consider removing all the helper modules.